src/pages/wallet-overview/wallet-overview.spec.ts

```diff
@@ -23,17 +40,222 @@ describe("WalletOverviewPage", () => {
         })
       ],
       providers: [
-        { provide: WalletService, useClass: MockWalletService },
-        { provide: NavController, useClass: MockNavController },
-        { provide: NavParams, useValue: new MockNavParams() }
+        { provide: WalletService, useValue: walletService },
+        { provide: NavController, useValue: navController },
+        { provide: NavParams, useValue: new MockNavParams() },
+        { provide: ToastController, useValue: toastController },
+        { provide: LoadingController, useValue: loadingController },
+        { provide: AlertController, useValue: alertController }
       ]
     }).compileComponents();
   }));
 
-  // beforeEach(() => {
-  //   fixture = TestBed.createComponent(WalletOverviewPage);
-  //   comp = fixture.componentInstance;
-  // });
+  beforeEach(() => {
+    fixture = TestBed.createComponent(WalletOverviewPage);
+    comp = fixture.componentInstance;
+  });
+
+  it("should create component", () => expect(comp).toBeDefined());
+
+  it("should have picked currency as undefined", () => {
```

Comment:
Right now we have two ways to check this initialization logic. I have created a single test named 'it should be initialised  correctly' and in this file we have a test for each individual property. How do we want to handle this?

Reply:
Discussed with @DanielLeushuis and we will use a single test to check for initialisation logic.